src/libraries/System.Diagnostics.Process/src/System/Diagnostics/DataReceivedEventArgs.cs

```diff
@@ -14,6 +41,16 @@ internal DataReceivedEventArgs(string? data)
             _data = data;
         }
 
+        /// <summary>Gets the line of characters that was written to a redirected <see cref="System.Diagnostics.Process" /> output stream.</summary>
+        /// <value>The line that was written by an associated <see cref="System.Diagnostics.Process" /> to its redirected <see cref="System.Diagnostics.Process.StandardOutput" /> or <see cref="System.Diagnostics.Process.StandardError" /> stream.</value>
+        /// <remarks>When you redirect the <see cref="System.Diagnostics.Process.StandardOutput" /> or <see cref="System.Diagnostics.Process.StandardError" /> stream of a <see cref="System.Diagnostics.Process" /> to your event handler, an event is raised each time the process writes a line to the redirected stream. The <see cref="System.Diagnostics.DataReceivedEventArgs.Data" /> property is the line that the <see cref="System.Diagnostics.Process" /> wrote to the redirected output stream. Your event handler can use the <see cref="System.Diagnostics.DataReceivedEventArgs.Data" /> property to filter process output or write output to an alternate location. For example, you might create an event handler that stores all error output lines into a designated error log file.
+        /// A line is defined as a sequence of characters followed by a line feed ("\n") or a carriage return immediately followed by a line feed ("\r\n"). The line characters are encoded using the default system ANSI code page. The <see cref="System.Diagnostics.DataReceivedEventArgs.Data" /> property does not include the terminating carriage return or line feed.
+        /// When the redirected stream is closed, a null line is sent to the event handler. Ensure your event handler checks the <see cref="System.Diagnostics.DataReceivedEventArgs.Data" /> property appropriately before accessing it. For example, you can use the static method <see cref="string.IsNullOrEmpty" /> to validate the <see cref="System.Diagnostics.DataReceivedEventArgs.Data" /> property in your event handler.
+        /// ## Examples
+        /// The following code example illustrates a simple event handler associated with the <see cref="System.Diagnostics.Process.OutputDataReceived" /> event. The event handler receives text lines from the redirected <see cref="System.Diagnostics.Process.StandardOutput" /> stream, formats the text, and writes the text to the screen.
+        /// [!code-cpp[Process_AsyncStreams#4](~/samples/snippets/cpp/VS_Snippets_CLR/process_asyncstreams/CPP/datareceivedevent.cpp#4)]
```

Comment:
Doesn't these have to be under a CDATA?

Reply:
Yes, I'm working on this 😄 . The tool needs to detect code examples or includes or hyperlinks, and in those cases, preserve the markdown.

Reply:
Fixed! The tool will avoid converting remarks to xmls if it finds examples, includes or hyperlinks.